calculator.html

```diff
@@ -0,0 +1,123 @@
+<html>
+<head>
+<meta charset="UTF-8">
+<title>Калькулятор</title>
+<script>
+ function control(testString) { 
+         testString = testString.replace(/\++/g, "+");
+         testString = testString.replace(/\--/g, "+");
+         testString = testString.replace(/\.\./g, ".");
+         testString = testString.replace(/[+-][-+]/g, "-");
+		 
+		
+	return testString;
+     }
+function Result(){
+var str = document.getElementById('text').value; 
+var inspection = str.indexOf ('/') + 1; 
+ 
+if (inspection != 0 ){ 
+alert("Делить на 0 нельзя");  

```

Comment:
This alert will fire each time when "/" symbol is found in calculator string. Also code doesn't stop execution inside of this if-statement.